auth0_flutter/example/ios/Tests/WebAuth/WebAuthSpies.swift

```diff
@@ -140,3 +142,26 @@ class SpyWebAuth: WebAuth {
         callback(logoutResult)
     }
 }
+
+class SpyWebAuthUserAgent: WebAuthUserAgent {
```

Comment:
This should be `MockWebAuthUserAgent` because it's not spying.

Reply:
Thanks. Should I move it into a new file as well or is it ok here?